samples/tab-add-in-combined/nodejs/teamsapp.local.yml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://developer.microsoft.com/json-schemas/teams-toolkit/teamsapp-yaml/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+provision:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: Discounts # Teams app name
+    writeToEnvironmentFile: # Write the information of created resources into environment file for the specified environment variable(s).
+      teamsAppId: TEAMS_APP_ID
+  - uses: script # Set TAB_DOMAIN for local launch
+    name: Set TAB_DOMAIN for local launch
+    with:
+      run: echo "::set-teamsfx-env TAB_DOMAIN=localhost:53000"
+  - uses: script # Set TAB_ENDPOINT for local launch
+    name: Set TAB_ENDPOINT for local launch
+    with:
+      run: echo "::set-teamsfx-env TAB_ENDPOINT=https://localhost:53000"
+  #- uses: teamsApp/validateManifest # Validate using manifest schema
+  #  with:
+  #    manifestPath: ./appPackage/manifest.json # Path to manifest template
+  - uses: teamsApp/zipAppPackage # Build Teams app package with latest env value
+    with:
+      manifestPath: ./appPackage/manifest.json # Path to manifest template
+      outputZipPath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+      outputJsonPath: ./appPackage/build/manifest.${{TEAMSFX_ENV}}.json
+  - uses: teamsApp/validateAppPackage # Validate app package using validation rules
+    with:
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
+  - uses: teamsApp/update # Apply the Teams app manifest to an existing Teams app in Teams Developer Portal. Will use the app id in manifest file to determine which Teams app to update.
+    with:
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
+
+deploy:
+  - uses: devTool/install # Install development tool(s)
+    with:
+      devCert:
+        trust: true
+    writeToEnvironmentFile: # Write the information of installed development tool(s) into environment file for the specified environment variable(s).
+      sslCertFile: SSL_CRT_FILE
+      sslKeyFile: SSL_KEY_FILE
+
+  - uses: cli/runNpmCommand # Run npm command
```

Comment:
Install NPM automatically. 

**Reference -**

# Run npm command
  - uses: cli/runNpmCommand
    with:
      workingDirectory: .
      args: install --no-audit

   # Run npm command
  - uses: cli/runNpmCommand
    with:
      workingDirectory: ./tab
      args: install --no-audit

   # Run npm command
  - uses: cli/runNpmCommand
    with:
      workingDirectory: ./add-in
      args: install --no-audit

Reply:
Will cover in different new PR

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965/files

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965